consumption.lua

```diff
@@ -1,4 +1,5 @@
 local Constants = require("constants")
+local City = require("city")
```

Comment:
i've tried this too today, but it leads to recursion-to-death because `city` also requires `consumption` :(

Reply:
I was expecting that. Will move the special buildings function into a util class.